grommet/grommet
# Max Size prop added, story changes, and docs updated for new prop

#### What does this PR do?
Adds the `maxSize` prop to `FileInput`, allowing users to create a limit on how large the files given to `FileInput` can be.

#### Where should the reviewer start?
- Check out the `maxSize` prop at  `index.d.ts` file for `FileInput` and its `doc.js`
- `yarn storybook` and navigate to FileInput's `Multiple` or `Simple` stories
- (Work in Progress): see how `maxSize` can be used in the code snippets of those stories and how its error messages work

#### What testing has been done on this PR?
Manual testing

#### How should this be manually tested?
- `yarn storybook` and navigate to FileInput's `Multiple` or `Simple` stories
- Add a file that is bigger than `maxSize` on the story
- (Work In Progress): see the error that pops up when `maxSize` is exceeded

#### Any background context you want to provide?

#### What are the relevant issues?
Issue #5320 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
Yes, updates to adding `maxSize` prop to `FileInput`

#### Should this PR be mentioned in the release notes?
No

#### Is this change backwards compatible or is it a breaking change?
Will be backwards compatible
